src/main/java/marquez/service/DatasourceService.java

```diff
@@ -0,0 +1,43 @@
+package marquez.service;
```

Comment:
minor: add license

Reply:
I'll add the license.
We should consider making this a build rule. Spotless has a way of helping with this:
https://github.com/diffplug/spotless/tree/master/plugin-gradle#license-header-options

Reply:
Please see PRs #394, #387, and #384, where this is fixed.